README.md

```diff
@@ -1,19 +1,52 @@
 # dojot History service
 
 This service handles all operations related to persisting and retrieving historical device
-data.
+data. For more information about the usage, check the [API documentation](https://dojot.github.io/history/apiary_latest.html).
 
-## Installation
+## About History
 
-As this service is written in Python 2.7, it is recommended that all packages are
-installed within a virtual environment. This can be created by:
+The history service is used when a device historical data needs to be retrieved, but it can also be utilized via HTTP request 
+employing different filters, like deviceId, number of entries to be listed and starting date, as in the following example:
+
+```shell
+http://host:port/device/{device_id}/history?lastN={lastN}&attr={attr}&dateFrom={dateFrom}&dateTo={dateTo}
+```
+
+### Dependencies
+- [falcon](https://falconframework.org/)
+- [gunicorn](https://gunicorn.org/)
+- [gevent](http://www.gevent.org/)
+- [python-dateutil](https://pypi.org/project/python-dateutil/1.4/)
+
+
+### Configuration
+
+In order to properly start History, the following variables must be set. All
+values are the default ones, and they need to be modified to the respective services' address.
+
+Key                         | Purpose                                                      | Default Value
+----------------------------|--------------------------------------------------------------|----------------
+HISTORY_DB_ADDRESS          |History database's address                                    |"mongodb"
+HISTORY_DB_PORT             |History database's port                                       |27017
+HISTORY_DB_REPLICA_SET      |History database's replica set address                        |None
+HISTORY_DB_DATA_EXPIRATION  |Seconds before removing an entry in MongoDB                   |604800
```

Comment:
I think this variable is not available in this service, can you check it, please